packages/common/src/ui/components/inputs/TextInput/InputWithLabelRow.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { FC, ReactNode } from 'react';
+import { Grid, FormLabel } from '@mui/material';
+import { BasicTextInput } from './BasicTextInput';
+
+interface InputWithLabelRowProps {
+  Input: ReactNode;
+}
+
+export const InputWithLabelRow: FC<InputWithLabelRowProps> = ({
+  Input = <BasicTextInput />,
+}) => {
+  return (
+    <Grid display="flex" alignItems="center" gap={1}>
+      <FormLabel sx={{ fontWeight: 'bold' }}>Customer name:</FormLabel>
```

Comment:
was going to point out that it needs a translation, but then noticed this is a shared component.. so will need the label to be passed in